### ISSUE TITLE
Bug with photo and text

### ISSUE THREAD

**Issue by vinay7722** [incubateind/gdtc-Day2#36]
![image](https://user-images.githubusercontent.com/60025240/107145464-9eb2d480-6967-11eb-95a7-15fc9c4b97bd.png)
icon size is not in perfact size
![image](https://user-images.githubusercontent.com/60025240/107145489-dc176200-6967-11eb-91bf-75afbedb62b6.png)
padding margin are not set

**Comment by rexyagami** [comment]
@vinay7722 yes, you have to fix these bugs and achieve the final design and then generate the PR.
